Vagrantfile

```diff
@@ -9,8 +9,8 @@ Vagrant.configure("2") do |config|
     config.vm.box = "puppetlabs/ubuntu-16.04-64-nocm"
 
     # VM: "local-sympa"
-    config.vm.define "local-sympa-ubuntu16" do |conf|
-        conf.vm.hostname = "local-sympa-ubuntu16"
+    config.vm.define "local-sympa" do |conf|
+        conf.vm.hostname = "local-sympa"
         # Give it a fixed IP
         conf.vm.network "private_network", ip: "192.168.66.67", :netmask => "255.255.255.0"
```

Comment:
why is there a hardcoded ip in here? I guess it makes some sense for Vagrant but is there a possibility of avoding it?

Reply:
It's only for usage with Vagrant indeed: we need to give the virtual server an Ip in order to know where it will answer. That' way, we know how to setup our /etc/hosts to be able to contact the server running locally.
I think the original author put it in the Vagrantfile because it is the only context where we setup IP from the project.
The thing is: before running Vagrant, we don't have an inventory. And we can't run the playbook without the inventory. So I really don't know how to use, e.g. the environment, to setup the IP.
Any suggestion welcome, though.